GUI/GUI.csproj

```diff
@@ -484,6 +484,9 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Resource>
   </ItemGroup>
+  <ItemGroup>
+    <Resource Include="Icons\proxl-ms.bmp" />
```

Comment:
add copy to output directory
![image](https://user-images.githubusercontent.com/16841846/69552454-19125700-0f64-11ea-956d-f3d4f271a939.png)

Also. please move it up one spot in the list